docs/Gemfile.lock

```diff
@@ -26,7 +26,7 @@ GEM
       liquid (~> 4.0)
       mercenary (>= 0.3.6, < 0.5)
       pathutil (~> 0.9)
-      rouge (>= 3.0, < 5.0)
+      rogue (>= 3.0, < 5.0)
```

Comment:
Do we know why this has changed? Is this an actual change or `codespell`?

Reply:
😱 I'm going blind: I thought this was just whitespace.
Good catch.

Yeah I'm 👎  on `codespell` in pre-commit. I didn't actually realise we had added it.

Reply:
Fixing now!

Reply:
And it was @dstansby

Reply:
#207